README.md

```diff
@@ -76,6 +79,16 @@ Which gives:
   * [ADDED, CHANGED] add model training process as a jupyter notebook (1c57956 by Runzhou Li (woozyking))
 ```
 
+
+### Tag
+
+Recently added, the sub-command `tag` identifies the version inside the commit messages and can either create a local tag with the commit version as its name or make a github release using the `--github` flag
```

Comment:
i think it's safe to assume the next version is 3.1 so we can say `Since v3.1.0, the sub-command ...`